packages/vsx-registry/src/browser/vsx-extensions-model.ts

```diff
@@ -212,7 +217,10 @@ export class VSXExtensionsModel {
 
     protected async refresh(id: string): Promise<VSXExtension | undefined> {
         try {
-            const data = await this.api.getExtension(id);
+            const data = await this.api.getLatestCompatibleExtensionVersion(id);
```

Comment:
it does only one request per an extension right?

Reply:
@akosyakov the current open-vsx API does not support getting for all versions of an extension it's corresponding `engines` therefore multiple requests are needed at the moment. I'm trying to see if there's a way to minimize the number of requests.